aspose/src/main/java/nl/nn/adapterframework/extensions/aspose/AsposeFontManager.java

```diff
@@ -15,9 +15,7 @@
 */
 package nl.nn.adapterframework.extensions.aspose;
 
-import java.awt.Font;
-import java.awt.FontFormatException;
-import java.awt.GraphicsEnvironment;
+import java.awt.*;
```

Comment:
Beware of `*` imports!

Reply:
Yeah, I saw this one too and fixed it but while committing, IJ brought it back 👎 I don't understand, because the settings on imports are on 9999+ 
Just wondering why he it ignorant here. This never happened before on my previous commits.

Reply:
Fixed by changing the `.editorconfig` file.